src/api.ts

```diff
@@ -35,6 +37,32 @@ export const bas = {
         return ActionsController.getAction(actionId);
     },
 
+    async getParameter(parameterName: string) : Promise<string | null> {
```

Comment:
why is it `string | null` is it not more appropriate to return `string | undefined` 

WDYT?

Reply:
@idoprz asked it

Reply:
@idoprz WDYT null or undefined?

Reply:
This is from @idoprz 
If the user tries to get value of key that does not exist it returns null. If running in VSCode all keys will return null as value for now.

Reply:
I talked to Ido and it doesn't matter for him null or undefined. 
I do love null, because undefined can come from some bug and null is more sure, but can change if you are sure it is better.